### DISCUSSION TITLE
How do I "fullscreen" the width of my dialog from q-select?

### DISCUSSION THREAD

**Discussion by codeblues08** [quasarframework/quasar#11083] [General - Components / Directives / etc] [answered]
I'm currently using the `q-select__dialog` class to customize my dialog and I can't figure out how to make my dialog width kind of like full screen (i don't know the right term). I tried setting the width to 100% but the width is still not fullscreen. Any idea how I should go  about this?

codepen link: [https://codepen.io/codeblues_/pen/YzxNZNr ](https://codepen.io/codeblues_/pen/YzxNZNr)

**Comment by pdanpdan** [comment]
Please make a codepen so we can play with it

**Reply by codeblues08** [reply]
hi, I modified my post and added the link. here it is https://codepen.io/codeblues_/pen/YzxNZNr thank you!

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/RwZKVqb

`max-width` to override default width limits

**Reply by codeblues08** [reply]
thank you! this worked!
